task/01-strings-tasks.js

```diff
@@ -130,7 +127,7 @@ function repeatString(value, count) {
  *   'ABABAB','BA' => 'ABAB'
  */
 function removeFirstOccurrences(str, value) {
-    throw new Error('Not implemented');
+    return str.slice(0,str.indexOf(value)) + str.slice(str.indexOf(value)+value.length, str.length);
```

Comment:
Please, try to follow these rules https://javascript.info/coding-style  (spaces between operators particularly)